.github/workflows/cleanup-preview.yml

```diff
@@ -0,0 +1,44 @@
+name: Cleanup Preview
+on:
+  pull_request:
+    branches: [main]
+    types: [closed]
+
+jobs:
+  cloud-login:
+    if: ${{ github.repository_owner == 'fermyon' }}
+    uses: "fermyon/actions/.github/workflows/auth.yml@main"
```

Comment:
nit: should we eventually pin the action versions (here and `rajatjindal/fermyon-preview-deploy` below) to tags as opposed to `main`?